source/features/git-checkout-pr.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import ClippyIcon from 'octicon/clippy.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCurrentBranch} from '../github-helpers';
+
+function handleMenuOpening({delegateTarget: dropdown}: delegate.Event): void {
+	dropdown.classList.add('rgh-git-checkout'); // Mark this as processed
+	const [, user, repository] = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname.split('/', 3);
+	const isLocalPr = select('.user-select-contain.head-ref a')!.childElementCount === 1;
+	const tabContainer = select('tab-container', dropdown)!;
+	tabContainer.style.width = 'fit-content';
+	select('.UnderlineNav-body', tabContainer)!.append(
+		<button
+			name="type"
+			type="button"
+			role="tab"
+			aria-selected="false"
+			className="UnderlineNav-item flex-1 btn-link"
+		>
+			Git Checkout
+		</button>
+	);
+
+	tabContainer.append(
+		<div hidden role="tabpanel" className="p-3">
+			<p className="text-gray text-small">
+				Run in your project repository
+			</p>
+			<div className="copyable-terminal">
+				<div className="copyable-terminal-button">
+					<clipboard-copy
+						className="btn btn-sm zeroclipboard-button"
+						role="button"
+						for="rgh-checkout-pr"
+						aria-label="Copy to clipboard"
+						data-copy-feedback="Copied!"
+					>
+						<ClippyIcon/>
+					</clipboard-copy>
+				</div>
+				<pre
+					id="rgh-checkout-pr"
+					className="copyable-terminal-content"
+				>
+					<span className="user-select-contain">
+						{isLocalPr ? '' : `git remote add ${user} ${location.origin}/${user}/${repository}.git
+`}
```

Comment:
I don't like what I did here. I needed there to be a new AND when the user clicks the copy button there should be a new line.

Reply:
Use `\n` instead of wrapping, like in the olden times without template literals

Reply:
```suggestion
						{isLocalPr ? '' : `git remote add ${user} ${location.origin}/${user}/${repository}.git\n`}
```

Reply:
🤦